src/main/java/core/basesyntax/service/impl/GreatReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.GreatReportService;
+import java.util.HashMap;
+
+public class GreatReportServiceImpl implements GreatReportService {
+    private static final String SEPARATOR_IN_LINE = ", ";
+    private static final StringBuilder stringBuilderReport = new StringBuilder();
```

Comment:
Report can be a variable of method
```suggestion
    private static final StringBuilder report = new StringBuilder();
```